separate_instance/docker-compose.override.yml

```diff
@@ -0,0 +1,130 @@
+version: "3.4"
+
+services:
+
+#  ########################################################
+#  # FRONTEND #############################################
+#  ########################################################
+#  frontend:
+#    # name the image so that it cannot be found in a DockerHub repository, otherwise it will not be built locally from the 'dockerfile' but pulled from there
+#    image: gradido/frontend:local-development
+#    build:
+#      target: development
+#    environment:
+#      - NODE_ENV="development"
+#      # - DEBUG=true
+#    volumes:
+#      # This makes sure the docker container has its own node modules.
+#      # Therefore it is possible to have a different node version on the host machine
+#      - frontend_node_modules:/app/node_modules
+#      # bind the local folder to the docker to allow live reload
+#      - ./frontend:/app
+#
+#  ########################################################
+#  # ADMIN INTERFACE ######################################
+#  ########################################################
+#  admin:
```

Comment:
no need to comment this out you can start stuff specifically `docker-compose up mariadb database backend`

Reply:
yes you're right, but i had trouble with some modul-dependencies, which cause starting modules automatically, but i just not needed them for my scenario.
Commenting them out was the easiest solution...